<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <node pkg="river" type="river_eval" name="river_eval_node" output="screen">
    </node>
</launch>